app/(home)/page.tsx

```diff
@@ -5,11 +5,13 @@ import Prizes from '@/app/(home)/Prizes';
 
 
 export default async function Home() {
+    // backdrop-blur is slow on FF
+    const maybeBlur = navigator.userAgent.search("Firefox") ? "" : "backdrop-blur-sm";
```

Comment:
Pretty sure this won't work as `navigator` is client-side only, and this is server-side code. We'd have to add an effect that parses this on the client, but atp I don't think its worth it.